external/mainnet/TToken.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Here it is: https://github.com/threshold-network/solidity-contracts/blob/main/deployments/mainnet/T.json

Reply:
Added as part of [82173bc](https://github.com/keep-network/coverage-pools/pull/203/commits/82173bc726c4622bda153c68c16c9aade559484e)